daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md

```diff
@@ -128,6 +134,42 @@ curl -X POST http://localhost:3500/v1.0/publish/myPulsar/myTopic?metadata.delive
       }'
 ```
 
+### OIDC Authentication
+
+Since `v3.0`, [Pulsar supports OIDC authentication](https://pulsar.apache.org/docs/3.0.x/security-openid-connect/).
```

Comment:
@JoshVanL would the changes we made support OAuth2 (without OIDC) too? https://pulsar.apache.org/docs/3.0.x/security-oauth2/

Reply:
No idea, we can add a certification test to see if it would work.